### ISSUE TITLE
Code for measuring dataset performance (mIoU)

### ISSUE THREAD

**Issue by fredpeertje** [intel-isl/Open3D-ML#229]
Is the code available for measuring the performance? Like using the mean intersection-over-union (mIoU) over all classes, as mentioned on the model_zoo page.

In the file `ml3d/datasets/utils/dataprocessing.py` it looks like this is calculated using confusion matrices.

**Comment by sanskar107** [comment]
These metrics are defined in `ml3d/torch/modules/metrics/` and `ml3d/tf/modules/metrics/`

**Comment by fredpeertje** [comment]
Oke, it looks like these metrics are only used in the training part. (line 384 in ml3d/torch/pipelines/semantic_segmentation.py). Is the code also available that you used for evaluating the [trained model](https://storage.googleapis.com/open3d-releases/model-zoo/kpconv_parislille3d_202011241550utc.pth) on the validation set of for example Paris-Lille3D?
